src/Endpoints/Indexes.php

```diff
@@ -152,49 +157,26 @@ public function fetchInfo(): self
         return $this->fill($response);
     }
 
-    public function update($body): self
+    public function update($body): array
     {
-        $response = $this->http->put(self::PATH.'/'.$this->uid, $body);
-
-        return $this->fill($response);
+        return $this->http->put(self::PATH.'/'.$this->uid, $body);
     }
 
     public function delete(): array
     {
         return $this->http->delete(self::PATH.'/'.$this->uid) ?? [];
     }
 
-    // Updates
+    // Tasks
 
-    public function getUpdateStatus($updateId): array
+    public function getTask($uid): array
     {
-        return $this->http->get(self::PATH.'/'.$this->uid.'/updates/'.$updateId);
+        return $this->tasks->getIndexTask(self::PATH.'/'.$this->uid, $uid);
```

Comment:
An inspiration for this could be the `stats` route that is also used in both client and index.